packages/lit-dev-content/site/tutorials/content/svg-templates/03.md

```diff
@@ -0,0 +1,98 @@
+### Overview
+
+This is chapter of the demo covers the basics of `<defs>` and `<use>` in
+SVG including how to:
+
+- define a group in defs
+- create a group of groups with `<use>`
+- create clip path
+- apply clip paths to a group
+
+### Learn
+
+The tile in this demo will be a series of transformed texts.
+
+Clip paths can be used to corral a group of visual elements. In SVG
+a clip path is defined with the `<clipPath>` element. It will take the
+shape of its child elements.
+
+In the example below, a clip path will display contents within a 
+rectangle with an id of `rect-clip`.
+
+```TS
+const helloClipPath = svg`
+	<clipPath id="rect-clip">
+		<rect width="200" height="200"></rect>
+	</clipPath>
+`;
+```
+
+In pattern making, a _tile_ is a polygon that can be repeated across
+a two dimensional plane. This demo uses a `<clip-path>` to create a
+tile containing five groups of rotated text.
+
+In the example below a 300px black square will be clipped into visually
+looking like a 200px black square.
+
+```TS
+const helloTile = svg`
+	<g clip-path="url(#rect-clip)">
+		<rect width="300" height="300" fill="#000"></react>
+	</g>`;
+```
+
+### Apply
+
+First, create a clip path composed of a 200px square.
```

Comment:
make sure you mention this is to create the tile

Reply:
done!